src/main/java/com/rakovets/course/java/core/practice/enum_types/EnumTypes/Year.java

```diff
@@ -0,0 +1,9 @@
+package com.rakovets.course.java.core.practice.enum_types.EnumTypes;
+
+public class Year {
+    Season seasonOfTheYear;
```

Comment:
Use `private` by default

Reply:
I made it public, since otherwise it is not visible in class EnumTypes